test-data/unit/semanal-errors.test

```diff
@@ -940,14 +940,6 @@ from typing import Generic
 class A(Generic[int]): pass # E: Free type variable expected in Generic[...]
 [out]
 
-[case testInvalidTypeWithinNestedGenericClass]
-from typing import Generic, TypeVar
-T = TypeVar('T')
-class A(Generic[T]):
-    class B(Generic[T]): pass \
-          # E: Free type variable expected in Generic[...]
```

Comment:
This appears in PEP 484 as an example of an error. I think it should still be prohibited.